tests/unit/models/test_dashboard_elasticsearch_document.py

```diff
@@ -15,14 +15,26 @@ def test_to_json(self):
                                        dashboard_name='test_dashboard_name',
                                        description='test_description',
                                        product='mode',
+                                       cluster='gold',
                                        dashboard_group_description='work space group',
+                                       query_names=['query1'],
+                                       group_url='mode_group_url',
+                                       url='mode_report_url',
+                                       uri='mode_report_uri',
```

Comment:
Suggestion: How about using value like "mode_dashboard://gold.cluster/dashboard_group/dashboard" so that other will understand what this would look like?